backend/test/card_game/api_winner_test.clj

```diff
@@ -0,0 +1,44 @@
+(ns card-game.api-winner-test
+  (:require [expectations.clojure.test :refer :all]
+            [card-game.api.base :as api]
+            [configs :as configs]))
+
+(defexpect win-conditions
```

Comment:
All these three expectations could be pretty much two-liners if we simply make ourselves a helper function above this
```clojure
(defn play-all-cards
  [player1 player2 strategy1 strategy2]
     ...
```

Reply:
I thought about something like that, but wanted to keep the `helper/end-game` as simple as possible. If you think it's worth I'll work something out.

Reply:
`helper/end-game` could be redefined to use this function internally, too, so it wouldn't really add much complication

Reply:
This is what I meant:

```clojure
(defn play-a-game-helper
  [strategy1 strategy2]
  (loop [game (api/create-game)
         opponent (api/add-player (:game-id game))
         iteration (count (configs/ini-hand))]
      (if (= 0 iteration)
        (api/get-game (:game-id game) (:player-id game))
        (recur
          (api/play-card-as-player (:game-id game) (:player-id game) 0 (strategy1 iteration))
          (api/play-card-as-player (:game-id game) (:player-id opponent) 0 (strategy2 iteration))
          (dec iteration)))))
    

(defexpect win-conditions
  ; Both stack all cards on one row -> tie
  (expect
    #(= :tie (:winner %))
    (play-a-game-helper
      (fn [i] 0)
      (fn [i] 0)))

  ; Opponent stacks all cards on one row and I spread -> I win
  (expect
    #(= :me (:winner %))
    (play-a-game-helper
      (fn [i] (mod i 4))
      (fn [i] 0)))

  ; I stack all cards on one row and opponent spreads -> Opponent wins
  (expect
    #(= :opponent (:winner %))
    (play-a-game-helper
      (fn [i] 0)
      (fn [i] (mod i 4)))))
```